.github/workflows/build.yml

```diff
@@ -49,10 +50,21 @@ jobs:
           fetch-depth: 1
 
       - name: Configure build - sanitizers ${{ matrix.sanitizers }}
-        run: cmake --preset ci
+        run: cmake --preset ci-${{ matrix.features }}
 
       - name: Build ${{ matrix.build-type }} with sanitizers set ${{ matrix.sanitizers }}
-        run: cmake --build --preset ci --config ${{ matrix.build-type }} -j $(nproc)
+        run: cmake --build --preset ci-${{ matrix.features }} --config ${{ matrix.build-type }} -j $(nproc)
 
       - name: Test ${{ matrix.build-type }} with sanitizers set ${{ matrix.sanitizers }}
-        run: ctest --preset ci --build-config ${{ matrix.build-type }}
+        run: ctest --preset ci-${{ matrix.features }} --build-config ${{ matrix.build-type }}
+
+      - name: Test that vast-front accepts option to produce SARIF files
```

Comment:
Make this normal `lit` test, so it is also checked when some runs tests locally.

You can set there required feature `sarif` so it will run only when `vast` is built with `sarif`.